scrapers/EliteBabes.yml

```
@@ -0,0 +1,50 @@
+name: EliteBabes
+performerByName:
+  action: scrapeXPath
+  queryURL: https://www.elitebabes.com/?s={}
+  scraper: performerSearch
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - www.elitebabes.com/model/
+    scraper: performerScraper
+
+xPathScrapers:
+  performerSearch:
+    performer:
+      Name: //article[@id='content']/ul[@class='gallery-a b d']/li/a[contains(@href,"model/")]/span/text()
+      URL:
+        selector: //article[@id='content']/ul[@class='gallery-a b d']/li/a[contains(@href,"model/")]/@href
+
+  performerScraper:
+    common:
+      $infoPiece: //div/ul[@class='list-aside hidden']/li/span
+    performer:
+      Name: //div/h2[@class='fn']/text()
+      Birthdate:
+        selector: $infoPiece[text() = 'Born:']/../text()
+      Gender:
+        fixed: "Female"
+      Height:
+        selector: $infoPiece[text() = 'Height:']/../text()
+        postProcess:
+          - replace:
+              - regex: .*\((\d+)cm\)
```

```suggestion
              - regex: .^(\d+).*
```

There is a wide variety of height formats on the site: Found examples of all of these:
```
152 cm
175 cm, 5 ft 9 in cm
172
```
https://www.elitebabes.com/model/airu-kaname/
https://www.elitebabes.com/model/alisa-i/
https://www.elitebabes.com/model/mila-azul/

---

Thanks